### ISSUE TITLE
Change organization and project name expansion

### ISSUE THREAD

**Issue by hammer** [sgkit-dev/sgkit#1167]
We [decided](https://github.com/pystatgen/sgkit/discussions/1166) to rename the organization to `sgkit-dev` and to expand `sgkit` to "Scalable genetics toolkit". We should make the corresponding changes and make sure all project resources work.

**Comment by jeromekelleher** [comment]
Things that might break:

- documentation hosted on GitHub pages: https://pystatgen.github.io/sgkit/latest/
- Workflows (do a grep for pystatgen in this repo?)
- Conda-package?

**Comment by benjeffery** [comment]
https://docs.github.com/en/organizations/managing-organization-settings/renaming-an-organization

**Comment by hammer** [comment]
Also relevant https://github.com/isaacs/github/issues/492

**Comment by hammer** [comment]
Okay going to try to change the org name now, let's see how this goes...

**Comment by hammer** [comment]
...and quickly encountered an issue. After the org rename, my plan was to create a new organization `pystatgen`, add a repo `sgkit`, and set up GitHub Pages to redirect to the `sgkit-dev/sgkit` site. Unfortunately GitHub seems to have retired this repository name:

<img width="575" alt="Screenshot 2024-03-28 at 9 06 00 AM" src="https://github.com/sgkit-dev/sgkit/assets/15233/015966b2-8a92-4a80-a74a-970233da7e7e">

According to [StackOverflow](https://stackoverflow.com/questions/62298084/the-repository-name-has-been-retired-and-cannot-be-reused), I can contact GitHub support to get this repository name unretired. I'll do that now.

If it takes longer than 24 hours to unretire the repo, I'll change the org name back for now. Let me know if anyone has concerns about our docs being broken longer than that...

**Comment by hammer** [comment]
Okay I've got a ticket filed with GitHub support, I expect this URL only works for me but for my own reference I'll record it here: https://support.github.com/ticket/personal/0/2699614

**Comment by hammer** [comment]
Now to change `pystatgen` to `sgkit-dev` in the repo, I got hits in these files. I think a find and replace should be fine. I'm wondering if we'll need to update our repo with some of the services we use such as Mergify and PyPI?

### Mergify
.mergify.yml

### Markdown docs
CONTRIBUTING.md
GOVERNANCE.md
README.md

### Setup
setup.cfg

### Scripts
.github/scripts/test_sgkit_bgen.py
.github/scripts/test_sgkit_plink.py
.github/scripts/test_sgkit_vcf.py

### Workflows
.github/workflows/benchmark.yml
.github/workflows/build.yml
.github/workflows/check-docs.yml
.github/workflows/docs.yml
.github/workflows/upstream.yml
.github/workflows/validation.yml
.github/workflows/wheels.yml
.github/workflows/windows.yml

### Benchmarks
benchmarks/asv.conf.json

### Docs
docs/conf.py
docs/contributing.rst
docs/how_do_i.rst
docs/user_guide.rst
docs/_static/switcher.json
docs/examples/gwas_tutorial.ipynb:
docs/examples/relatedness_tutorial.ipynb
docs/news/introducing_sgkit.md

### Code
sgkit/io/vcfzarr_reader.py
sgkit/stats/pca.py

### Tests
sgkit/tests/test_ld.py
sgkit/tests/test_pc_relate.py
sgkit/tests/test_pedigree.py
sgkit/tests/test_regenie.py
sgkit/tests/test_utils.py
sgkit/tests/io/plink/test_plink_reader.py
sgkit/tests/io/vcf/test_vcf_scikit_allel.py

### Validation
validation/gwas/method/pc_relate/Dockerfile
validation/gwas/method/regenie/sgkit_zarr.py
validation/gwas/method/regenie_loco_regression/GlowGR_continuous.ipynb

**Comment by hammer** [comment]
I went ahead and committed the find and replace directly at https://github.com/sgkit-dev/sgkit/commit/d09fe246fe37fa7e6a6381e4307b8ed24e5c8f6b.

**Comment by hammer** [comment]
According to https://whois.domaintools.com/pystatgen.org the registration for `pystatgen.org` has lapsed, so it's probably a good thing we're changing org names! I can't find where I registered that domain anyways.

I went ahead and bought `sgkit.dev`

**Comment by hammer** [comment]
Okay docs look good at https://sgkit-dev.github.io/sgkit/latest. I noticed that for some reason https://sgkit-dev.github.io/sgkit is redirecting to https://pystatgen.github.io/sgkit/latest, and I'm not quite sure where that rule is encoded. It also seems that older versions of the docs are no longer available. I believe @tomwhite might know more about how our docs interact with GitHub Pages to figure this out?

**Comment by tomwhite** [comment]
> I noticed that for some reason https://sgkit-dev.github.io/sgkit is redirecting to https://pystatgen.github.io/sgkit/latest, and I'm not quite sure where that rule is encoded.

It's in the gh-pages branch: https://github.com/sgkit-dev/sgkit/blob/gh-pages/index.html

> It also seems that older versions of the docs are no longer available.

The version switcher at https://sgkit-dev.github.io/sgkit/latest/ doesn't seem to be appearing, although older versions do exist, e.g. https://sgkit-dev.github.io/sgkit/0.7.0/. However, the versions that they link to are wrong because they have the 
pystatgen.github.io. I'm not sure there's much we can do about that.

**Comment by jeromekelleher** [comment]
Awesome! Thanks for sorting this out @hammer

**Comment by jeromekelleher** [comment]
Given that we have the sgkit.dev domain, I wonder if it would be worth making our stable docs URL point there? So, rather than https://sgkit-dev.github.io/sgkit/ have something like https://sgkit.dev/sgkit/docs or something? It's a bit easier to remember, and would be easier to keep stable long term. We can still host using GitHub pages, it's effectively just a redirect. That's what we do with tskit.dev isn't it @benjeffery?

**Comment by hammer** [comment]
Ah yes in my reading I've seen comments about using a top level domain like that to avoid the situation we're in currently. To be honest managing DNS is not my favorite thing in the world. I will file it as an issue and come back to it a bit later?

Also they've un-retired `pystatgen/sgkit` thankfully so I will set up the redirects today when I have some time and close this out!

**Comment by hammer** [comment]
Okay I've resurrected `pystatgen/sgkit` and https://pystatgen.github.io/sgkit now redirects to https://sgkit-dev.github.io/sgkit.

One thing I did not think about: when we bring back this repository, we lose the automatic redirects that GitHub sets ups from all `https://github.com/pystatgen/sgkit/*` URLs to `https://github.com/sgkit-dev/sgkit/*` URLs.

This means links like https://github.com/pystatgen/sgkit/discussions will now 404 instead of redirecting to https://github.com/sgkit-dev/sgkit/discussions.

In short: we've gained a redirect for our Pages, but lost the redirect for our repo.

If everyone is okay with this tradeoff, I'm ready to close this out.

**Comment by jeromekelleher** [comment]
I'm good with that

**Comment by hammer** [comment]
Great filed https://github.com/sgkit-dev/sgkit/issues/1212 to follow up and will close this one out for now!
